### ISSUE TITLE
panos_mgtconfig not setting Timezone or MOTD

### ISSUE THREAD

**Issue by pmusolino-rms** [PaloAltoNetworks/ansible-pan#36]
Panos version: 8
ansible-pan==1.0.4
pan-python==0.12.0
pandevice==0.5.1
```
- name: Generic management settings
  panos_mgtconfig:
      ip_address: '{{ ip }}'
      username: '{{ user }}'
      password: '{{ password }}'
      timezone: 'UTC'
      login_banner: |
          +-------------------------------------------------------------+
          |               !!! Authorized access only !!!                |
          |     You are authorized to use this System for approved      |
          |     business purposes only. Use for any other purpose       |
          |     is prohibited.                                          |
          |                                                             |
          |     All transactional records generated by using this       |
          |       System are the property of ABC and may be             |
          |        used by ABC for any purpose. Authorized              |
          |     and unauthorized activities will be monitored.          |
          +-------------------------------------------------------------+

          +-------------------------------------------------------------+
          |     Disconnect IMMEDIATELY if you are not authorized        |
          |     For access to this system contact:                      |
          |                                                             |
          |                 email@example.com                           |
          +-------------------------------------------------------------+
      commit: true
```
Expected results:
deviceconfig section of config should contain timezone UTC and the above login-banner section
Ansible task should come back as changed

Actual results: 
No changes made, ansible command returns OK
```
TASK [pan-azure-security : Generic management settings] **************************************************************************************************************************************************************************
ok: [hostname.example.com]
```

**Comment by pmusolino-rms** [comment]
Debug mode

ok: [hostname.example.com] => {
    "changed": false,
    "failed": false,
    "invocation": {
        "module_args": {
            "api_key": null,
            "commit": true,
            "devicegroup": null,
            "dns_server_primary": null,
            "dns_server_secondary": null,
            "domain": null,
            "hostname": null,
            "ip_address": "hostname.example.com",
            "login_banner": "+-------------------------------------------------------------+\n|               !!! Authorized access only !!!                |\n|     You are authorized to use this System for approved      |\n|     business purposes only. Use for any other purpose       |\n|     is prohibited.                                          |\n|                                                             |\n|     All transactional records generated by using this       |\n|       System are the property of ABC and may be             |\n|        used by ABC for any purpose. Authorized              |\n|     and unauthorized activities will be monitored.          |\n+-------------------------------------------------------------+\n\n+-------------------------------------------------------------+\n|     Disconnect IMMEDIATELY if you are not authorized        |\n|     For access to this system contact:                      |\n|                                                             |\n|                 email@example.com                      |\n+-------------------------------------------------------------+\n",
            "ntp_server_primary": null,
            "ntp_server_secondary": null,
            "panorama_primary": null,
            "panorama_secondary": null,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "timezone": "UTC",
            "update_server": null,
            "username": "username"
        }
    },
    "msg": "okey dokey"
}

**Comment by ivanbojer** [comment]
Thank you for opening the issues.

There is a BUG in code that would not apply changes unless you change ntp server. It will be addressed in V1.0.5 and check has been checked in the develop branch already.
